python-sql-scripts/addCustomers.py

```diff
@@ -2,7 +2,15 @@
 import names
 import random
 import string
-from credentials import mysql_endpoint, username, password, database_name
+# from credentials import mysql_endpoint, username, password, database_name
+
+# print("hey")
+# print(password)
+
+mysql_endpoint='localhost'
+username='root'
+password="tatakae"
+database_name = 'testudo_bank'
```

Comment:
You shouldn't need to add these credentials again. They should be fetched from `credentials.py`.